python/lsst/meas/algorithms/convertRefcatManager.py

```diff
@@ -454,3 +454,35 @@ def gaiaFluxToFlux(flux, zeroPoint):
         result['phot_rp_mean_fluxErr'] = result['phot_rp_mean_flux'] / input['phot_rp_mean_flux_over_error']
 
         return result
+
+class ConvertGaiaXpManager(ConvertRefcatManager):
+    """Special-case convert manager to deal with refcats that have no MagErrors
+    """
+    def _getFluxes(self, inputData):
+        """Compute the flux fields that will go into the output catalog.
+
+        Parameters
+        ----------
+        inputData : `numpy.ndarray`
+            The input data to compute fluxes for.
+            from Gaia Collaboration, Montegriffo et al. 2022 flux units are  W/Hz/(m^2) 
+
+        Returns
+        -------
+        fluxes : `dict` [`str`, `numpy.ndarray`]
+            The values that will go into the flux and fluxErr fields in the
+            output catalog, units are nJy
+        """
+        result = {}
+        for item in self.config.mag_column_list:
+            
+            inFluxKey=item.replace('_','_flux_')
+            outFluxKey=item + '_flux'
+            
+            inFluxErrKey=item.replace('_','_flux_error_')
+            outFluxErrKey=item + '_fluxErr'
+            
+            result[outFluxKey] = (inputData[inFluxKey]*u.Watt/u.Hz/u.meter/u.meter).to(u.nJy)
+            result[outFluxErrKey] = (inputData[inFluxErrKey]*u.Watt/u.Hz/u.meter/u.meter).to(u.nJy)
```

Comment:
I think you need `to_value(u.nJy)` here to get a float array, instead of a Quantity array.